### ISSUE TITLE
[Bug]: Cannot open links in new tab

### ISSUE THREAD

**Issue by MarvinKlein1508** [microsoft/fluentui#36567]
### Component

Link

### Package version

3.1.0

### @microsoft/fast-element version

whatever the demos are using

### Environment

```shell
System:
    OS: Windows 11 10.0.26200
    CPU: (28) x64 Intel(R) Core(TM) i7-14700K
    Memory: 14.63 GB / 31.78 GB
  Browsers:
    Edge: Chromium (151.0.4129.78)
    Firefox Developer Edition: 154.0 - C:\Program Files\Firefox Developer Edition\firefox.exe
```

### Current Behavior

Nothing happens

### Expected Behavior

Both `fluent-link` and `fluent-anchor-button` should allow the user to open the link in a new tab by pressing the middle mouse button.

### Reproduction

https://storybooks.fluentui.dev/web-components/?path=/docs/components-link--docs

### Steps to reproduce

Click on the `Link` with your middle mouse button

### Are you reporting an Accessibility issue?

None

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by MarvinKlein1508** [comment]
@chrisdholt can you take a look? There are 2 open PRs for this :)
